packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2545,22 +2556,16 @@ class StackViewport extends Viewport implements IStackViewport {
    * @param colormap - The colormap data to use.
    */
   public setColormap(colormap: CPUFallbackColormapData): void {
-    if (this.useCPURendering) {
-      this.setColormapCPU(colormap);
-    } else {
-      this.setColormapGPU(colormap);
-    }
+    this.useCPURendering
```

Comment:
A better implementation is where the useCPURendering is called, just assign this.setColorMap = thisuseCPURendering ? this.setColormapCPU : this.setColormapGPU
Then you don't need the double indirection.  You can still declare setColormap as a public function, just assign the value once you know cpu/gpu usage.